app/dashboard/templates/profiles/tab_resume.html

```diff
@@ -0,0 +1,56 @@
+{% load i18n static %}
+
+
+    {% if not hidden and not profile.is_org and show_resume_tab %}
+    <div class="container tab4" >
+      <div id="resume-section">
+        <p>
+          <a href="/onboard/profile/?steps=job"> Edit this info
+          </a>
+        </p>
+
+        {% if not profile.show_job_status %}
+        <p>
+          <i class="fas fa-eye"></i> Only visible to you; Your information is private.
+        </p>
+        {% endif %}
+
+        <p>
+        Job Search Status: {{profile.job_status_verbose}}
+        </p>
+
+        <p>
+        Looking for work in Keywords:
+          <ul> 
+            {% for keyword in profile.keywords %}
+              <li>
+                <a href="/explorer?keywords={{keyword}}">{{keyword}}</a>
+              </li>
+            {% endfor %}
+          </ul>
+        </p>
+
+        <p>
+          Locations: 
+          <ul> 
+            {% for loc in profile.job_location %}
+              <li>
+                <a href="{{loc.url}}">{{loc.name}}</a>
```

Comment:
**XSS here**. When saving the profile, it is possible to intercept the request and change the parameters of _job_location_ url to contain javascript code. I just did it with my profile. 

2 Problems:
  - There is no validation of data entered from maps when saving the profile. It is possible to enter payloads.
  - There is no validation of data retrieved from job_location. It is possible to load and run payloads.

Possible consequences include stealing data, taking over the account, and downloading private keys containing tips.

![Zrzut ekranu z 2019-08-29 13-09-55](https://user-images.githubusercontent.com/18102267/63935845-a118bb80-ca5e-11e9-952f-1c07ab9a794a.png)